CHANGELOG.asciidoc

```diff
@@ -95,6 +95,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 - The `elasticsearch/node` metricset now reports the Elasticsearch cluster UUID. {pull}8771[8771]
 - Add service.type field to Metricbeat. {pull}8965[8965]
 - Support GET requests in Jolokia module. {issue}8566[8566] {pull}9226[9226]
+- Add `os.name` field into libbeat add_host_metadat processor. {issue}8948[8948] {pull}9405[9405]
```

Comment:
@kaiyan-sheng The PR still seems to contain the old changelog?

Reply:
oh man... sorry!! I somehow reversed it...